### PR TITLE
 imageSrc field is commented for solving this issue and can be removed

### LINKED ISSUE #9012: imageSrc in config.json while creating an app
### Issue Summary

While creating an application I found out that, imageSrc is still being added in the config.json file.

### Steps to Reproduce

1. Create an app with cli `yarn app-store create-template`
2. Fill in the details.
3. Check config.json file.

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Technical details

- [Recording](https://www.loom.com/share/43a6133247fa4fd59ca2a0545887da6d).
- Node.js version v16.20.0

### PR DESCRIPTION
**Pull request by sdey09**
## What does this PR do?

Removing the `imageSrc` field mentioned in #9012 

Fixes #9012 

**Environment**: Staging(main branch) / Production

## Type of change

- Commented the field `imageSrc`

## How should this be tested?
1. Create an app with cli yarn app-store create-template
2. Fill in the details.
3. Check config.json file.

image

### CODE REVIEW

**Comment by PeerRich** [comment]
@hariombalhara can review this better than me hehe

File: packages/app-store-cli/src/core.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export const BaseAppFork = {
       slug: slug,
       type: `${slug}_${category}`,
       // TODO: Remove usage of imageSrc, it is being used in ConnectCalendars.tsx. After that delete imageSrc in all configs and from here
-      imageSrc: `icon.svg`,
+      // imageSrc: `icon.svg`,
```
Comment (zomars) [inline packages/app-store-cli/src/core.ts:97]: Why comment it our instead of removing it?

Comment (Subhojit-Dey1234) [reply packages/app-store-cli/src/core.ts:97]: To be just sure about it. I thought something might break seeing the above comment.

Comment (Subhojit-Dey1234) [reply packages/app-store-cli/src/core.ts]: I have removed the field in the latest commit.

File: packages/app-store-cli/src/core.ts [hunk 2]
```diff
@@ -94,7 +94,6 @@ export const BaseAppFork = {
       slug: slug,
       type: `${slug}_${category}`,
       // TODO: Remove usage of imageSrc, it is being used in ConnectCalendars.tsx. After that delete imageSrc in all configs and from here
```
Comment (hariombalhara) [inline packages/app-store-cli/src/core.ts:96]:
```suggestion
```

**Review by hariombalhara — APPROVED** [review]
Thanks @Subhojit-Dey1234 🙏 There is no problem in removing it as cleanup has already been done as part of https://github.com/calcom/cal.com/pull/8479

**Pull request merged**